### PR TITLE
WIP fix peakachu dependencies

### PR DESCRIPTION
**Pull request by dmaticzka**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
CCI was someone stuck, lets try to re-trigger the test here.

File: recipes/peakachu/meta.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ requirements:
     - setuptools
     - "biopython >=1.69"
     - "matplotlib >=2.0.2"
-    - "pandas >=0.20.2"
+    - "pandas >=0.20.2 <=0.20.3"
```
Comment (mbargull) [inline recipes/peakachu/meta.yaml]: Should be `pandas >=0.20.2,<=0.20.3`, note the comma.

File: recipes/peakachu/meta.yaml [hunk 2]
```diff
@@ -32,7 +32,7 @@ requirements:
     - python
     - "biopython >=1.69"
     - "matplotlib >=2.0.2"
-    - "pandas >=0.20.2"
+    - "pandas >=0.20.2 <=0.20.3"
```
Comment (mbargull) [inline recipes/peakachu/meta.yaml]: Same as above.

**Comment by mbargull** [comment]
@dmaticzka, can you explain why that version restriction on `pandas` would be needed? There is no such restriction in the source at https://github.com/tbischler/PEAKachu/blob/v0.1.0/setup.py#L18.

**Comment by dmaticzka** [comment]
@mbargull it's a bug related to https://github.com/pandas-dev/pandas/issues/18021. pending resolution in either pandas or peakachu, peakachu will only work with either pandas version 0.20.2 or 0.20.3

**Comment by bgruening** [comment]
@dmaticzka I assume this one it not WIP anymore?

**Comment by mbargull** [comment]
@bgruening, there is an [open issue](https://github.com/tbischler/PEAKachu/issues/8) and a [commit](https://github.com/tbischler/PEAKachu/commit/ae64415ca887fce5a11bf848925cbbeb4adb4bfa) that's meant to fix that bug. If @dmaticzka can confirm this work, we could rather add a patch.

**Comment by dmaticzka** [comment]
in principle this should fix it, didn't stay up late for the circleci results to finally trickle in :)

this, the patch, or a new version for the patch would all work for me. i have to take care of a paper submission, so i can't check on that patch until end of next week.

**Comment by mbargull** [comment]
Ah, sorry, forgot to hit that "Approve" button. Thanks @dmaticzka!

**Pull request merged**